models/payment/src/payment-status/generator.ts

```diff
@@ -0,0 +1,14 @@
+import { Generator } from '@commercetools-test-data/core';
+import { TPaymentStatus } from './types';
+
+// https://docs.commercetools.com/api/projects/payments#paymentstatusdraft
+
+const generator = Generator<TPaymentStatus>({
+  fields: {
+    interfaceCode: null,
+    interfaceText: null,
+    state: null,
```

Comment:
This would be a full-blown `State` model, which is then transformed into a `Reference` for the REST model and the `stateRef` field of GraphQL.